input.go

```diff
@@ -18,6 +18,13 @@ import (
 	"github.com/hajimehoshi/ebiten/internal/ui"
 )
 
+// InputChars returns a slice of Unicode "printable" runes read from the keyboard.
+//
```

Comment:
Please add comments noting that this function is different from `IsKeyPressed` because `InputChars` returns actual input chars instead of physical (US) keys.